src/Plotly.NET.CSharp/Helpers.cs

```diff
@@ -15,7 +15,16 @@ static class Helpers {
         /// </summary>
         /// <param name="thing">the thing to turn into a FSharpOption</param>
         /// <returns>The original value wrapped as a FSharpOption</returns>
-        static internal Microsoft.FSharp.Core.FSharpOption<T> ToOption<T>(this T? thing) => thing is null ? Microsoft.FSharp.Core.FSharpOption<T>.None : new(thing);
+        static internal Microsoft.FSharp.Core.FSharpOption<T> ToOption<T>(this T? thing) {
+            if (EqualityComparer<T>.Default.Equals(thing, default(T)))
+            {
+                return FSharpOption<T>.None;
+            }
+            else
+            {
+                return new(thing);
+            }
+        }
```

Comment:
Why this change? `T?` is not nullable if `T` is a value type

Reply:
i dont remember the exact repro for this, but iirc without this default check i have gotten default values for non-nullable generic types instead of `None`

Reply:
Well if i revert this, this line gets me the following wrongly set defaults instead of `None`:

```csharp
Chart.Column<int,string,string>(
    values: new int [] { 3,4 },
    Keys: new string [] { "first", "second"}
)
```

```js
"base":0,"width":0,
```

which leads to an empty graph (width = 0). This is a problem only when a single value is set, collections do not have this default value problem. So the defaults that get omitted are `0`, `0.0`, `""`, and `false`, and that only when we are talking about generics. I would argue that this is better than charts that are invisible per default.